aws-redshift-clusterparametergroup/aws-redshift-clusterparametergroup.json

```diff
@@ -77,7 +77,9 @@
         "/properties/Description"
     ],
     "writeOnlyProperties": [
-        "/properties/Parameters"
+        "/properties/Parameters",
+        "/properties/Parameters/*/ParameterName",
```

Comment:
Adding `*/ParameterName` and `*/ParameterValue` because, for some reason, Contract Test V2 cannot digest nested property structure, meaning we need to list all the sub-elements down if necessary.